sqliter-driver/src/nativeCommonTest/kotlin/co/touchlab/sqliter/DatabaseConnectionTest.kt

```diff
@@ -255,17 +255,6 @@ class DatabaseConnectionTest {
 
         assertEquals(1, conn1.longForQuery("select count(*) from test"))
 
-        assertFails {
-            val connFail = man.surpriseMeConnection()
-            connFail.withTransaction {
-                it.withStatement("insert into test(num, str)values(?,?)") {
-                    bindLong(1, 232)
-                    bindString(2, "asdf")
-                    executeInsert()
-                }
-            }
-        }
```

Comment:
Removed this, as the only reason it was failing was b/c create callback was not being invoked after first new connection was created.